### PR TITLE
Sagemaker Compatibility

### PR DESCRIPTION
**Pull request by zhanghang1989**
*Description of changes:*

Many thanks to @tlienart !

Working with SageMaker example at https://github.com/zhanghang1989/AutoGluon-Docker

The change does not break any existing features.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: autogluon/scheduler/remote/remote.py [hunk 1]
```diff
@@ -68,6 +68,9 @@ def __init__(self, remote_ip=None, port=None, local=False, ssh_username=None,
         else:
             remote_addr = (remote_ip + ':{}'.format(port))
             self.service = start_service(remote_ip, port)
+            _set_global_remote_service(self.service)
+            import time
+            time.sleep(10)
```
Comment (Innixma) [inline autogluon/scheduler/remote/remote.py]: Can this be a lower value? What impact will this have on repeated HPO runs for many different models like in Tabular?

Comment (zhanghang1989) [reply autogluon/scheduler/remote/remote.py]: 1). This does not impact single machine training. 

2). The 10 seconds overhead is introduced only once at the starting of the remote nodes, which is usually acceptable.  There won't be any overhead during the job scheduling.

Comment (zhanghang1989) [reply autogluon/scheduler/remote/remote.py]: > Can this be a lower value? What impact will this have on repeated HPO runs for many different models like in Tabular?

There is no overhead for repeated run.

**Comment by szha** [comment]
Job PR-395-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-395/1/index.html

File: autogluon/task/object_detection/data_parallel.py [hunk 2]
```diff
@@ -53,6 +52,7 @@ def forward_backward(self, x):
             rcnn_l1_loss_metric = [[box_targets, box_masks], [box_pred]]
 
             if self.amp:
+                from mxnet.contrib import amp
```
Comment (Jerryzcn) [inline autogluon/task/object_detection/data_parallel.py:55]: is there a specific reason for this?

**Pull request merged**